DELIVERY.README.md

```diff
@@ -1,5 +1,5 @@
-![](doc/images/CAW_logo.png)
-# Cancer Analysis Workflow Results Delivery 
+![](doc/images/Sarek_logo.png)
+# Sarek - Cancer Analysis Workflow Results Delivery 
```

Comment:
`Sarek Somatic` instead of `Sarek - Cancer Analysis Workflow` ?

Reply:
Merging it, I'll make this change in my next PR fixing issue #535

Reply:
OK, thought you will get more of these.